pom.xml

```diff
@@ -187,6 +187,7 @@
     <module>hazelcast</module>
     <module>zookeeper</module>
     <module>dynamodb</module>
+    <module>s3</module>
```

Comment:
I suggest rename to amazon-s3. It will be still short. Today S3 abbreviation is obvious, but in future who knows.